target/linux/ramips/dts/WIZFI630S.dts

```diff
@@ -0,0 +1,196 @@
+/dts-v1/;
```

Comment:
@7tobias Please can you consider `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` as in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?

> Having MIT as alternative is sometimes preferred by upstream maintainers
and allows sharing that simple code with other projects. We don't really
want multiple DTS versions for the same device.